apps/datahub/src/app/dataset-view/dataset-view.component.html

```diff
@@ -1 +1,4 @@
-<p>dataset-view works!</p>
+<div class="float-right w-96 h-96 m-3 border rounded overflow-hidden">
+  <gn-ui-data-preview-map></gn-ui-data-preview-map>
+</div>
+<gn-ui-metadata-full-view></gn-ui-metadata-full-view>
```

Comment:
`metadata-full-view` was the larger container, and NOT the information tab or whereever you put the abstract and the preview map.
We must define a clear treeview, let's discuss about that.
- metadata(dataset)-view (the whole view)
  - summary
    - metadata-info
    - dataset-preview
  - data table
  - map
  - exports
  - apis

or something like this.
In my PR, i put the tabs in `ui-metadata-full-view`